config/deploy/dev.rb

```diff
@@ -0,0 +1 @@
+server 'upaya-dev.18f.gov', roles: %w(web app db)
```

Comment:
I'll test that out, it may also work for me because I have my ssh config specifying the user for that hostname. I'll move it back to the env config if need be.